.github/workflows/windows.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         continue-on-error: true
         run: |
           cd _build/tests
-          make check TESTSUITEFLAGS="--jobs=$(($(nproc)+1))" || (echo "not all tests passed")
+          make check || (echo "not all tests passed")
```

Comment:
I added that because, from time to time, running the testsuite would last forever... as it is now happening for macos. The fact that `-j N` with `N > 1` makes it impossible to understand on which test the testsuite is blocking. Actually, it might even be the case that two tests ran in parallel are blocking on some shared ressource...

Reply:
I've seen "blocking" mostly on MSYS so far. Running "broken" versions of GnuCOBOL also led to a blocking WSL in the past (never happened on a real or virtual GNU/Linux environment, as far as I know).

I _think_ that this was not reproducible (the test often hangs at different places), with the exception of broken memory access where this blocking may happened in the same test (but then also in single-core testing).